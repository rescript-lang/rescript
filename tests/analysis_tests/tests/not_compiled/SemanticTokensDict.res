let d = dict{"foo": bar}
//^sem
