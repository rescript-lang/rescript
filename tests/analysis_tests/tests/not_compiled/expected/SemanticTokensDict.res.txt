{"data":[0,4,1,1,0,0,16,3,1,0]}
